apps/web/src/design-system/navigation/NavMenu.cy.tsx

```diff
@@ -0,0 +1,47 @@
+import { NavMenu } from './NavMenu';
+import { MemoryRouter as Router } from 'react-router-dom';
+import { TestWrapper } from '../../testing';
+
+it('should active when clicked menu item', () => {
+  const activeClass = 'mantine-g10fyh';
+  const notActiveClass = 'mantine-16w7c2e';
```

Comment:
Hey love the new component test, do mantine create those classes always with the same class name as above?

Reply:
HI @djabarovgeorge Yes. always the same. like styled component